aten/src/ATen/native/GatedLinearUnit.cpp

```diff
@@ -93,6 +94,49 @@ Tensor glu_jvp(
   return dglu;
 }
 
+Tensor glu_vjp(
```

Comment:
Doesn't `vjp` just imply backward? Maybe we should call this `glu_backward_jvp` instead?